lib/game.rb

```diff
@@ -1,5 +1,22 @@
 # frozen_string_literal: true
 
+# Main class for Tic Tac Toe game
 class Game
-  puts 'Welcome to Tic Tac Toe!'
+  
+  def initialize
+    @board = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+
+    @board_display = "#{@board[0]} | #{@board[1]} | #{@board[2]}\n-----------\n#{@board[3]} | #{@board[4]} | #{@board[5]}\n-----------\n#{@board[6]} | #{@board[7]} | #{@board[8]}"
+
+    @welcome_message = 'Welcome to Tic Tac Toe!'
+  end
+  
+  def run
+    print_to_console
+  end
+
+  def print_to_console
+    puts @welcome_message + "\n" + @board_display
+  end
+
```

Comment:
The game seems to have a lot of responsibilities.  What should the game be responsible for?  Currently, it's doing the following:

- Managing the board
- Formatting the board
- Outputting the board
- Executing the game

Reply:
I've been wondering when it's time to add a new class... The bullet points are helpful :) I'm guessing I might need one class for the first 3 bullet points (probably board), and then game could just be responsible for the last bullet point.

Reply:
You got it @karen-olson !

Reply:
For example, think of the outputting the board.  It could be a class to output something, anything you pass to it.  You could pass a formatted board to it, or any other string.  All it does is output.

Reply:
Oh!! My first thought when Damon said you could only use puts once was to basically make a custom "puts", but I thought that would be cheating haha. Now I understand what you mean by "what happens if puts changes".

Reply:
Actually rethinking this you don't need the managing the board bullet yet, since you're only displaying an empty board.